usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -3991,22 +3991,24 @@ lxpr_read_meminfo(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	 * model, so just inform the caller that no swap is being used.
 	 */
```

Comment:
can we get a comment on MemAvailable semantics here, namely that as it's just an estimate, we always return MemFree instead of attempting to assess reclaimable memory?

Reply:
yes, this is just an estimate to make happy applications that search for this procfs entry. 
We will always return free_mem, but actually the right value to return is ( total_mem - free_mem). 
If you agree I'll make that change and resubmit.

Reply:
I don't see why it would be total-free. As per the docs:

MemAvailable: An estimate of how much memory is available for starting new
              applications, without swapping. Calculated from MemFree,
              SReclaimable, the size of the file LRU lists, and the low
              watermarks in each zone.
              The estimate takes into account that the system needs some
              page cache to function well, and that not all reclaimable
              slab will be reclaimable, due to items being in use. The
              impact of those factors will vary from system to system.

So from that it should be at *least* MemFree, but potentially more. As we can't easily calculate the reclaimable amount, it seems like MemFree is the right value?

Reply:
Oh, ok so using free_mem as MemAvailable seems correct then!. No changes needed on this PR right ? maybe I'm getting confused.

Reply:
I was asking for a comment on the semantics to be put with the code.

Reply:
I'm sorry. what do you think on adding this comment ?

> MemAvailable entry is available since Linux Kernel +3.14, is an estimate of how much memory is available for starting new applications, without swapping. In lxbrand we will always return the available free memory as an estimate of this value.

Reply:
Sounds fine